ftw/simplelayout/profiles/default/portlets.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<portlets>
+    <assignment
+      manager="plone.rightcolumn"
+      category="content_type"
+      key="ftw.simplelayout.ContentPage"
```

Comment:
Disabling is possible by:
- Plone Control Panel -> Content types -> content specific portlets
- Blocking the portlet on a specific context


Context assignment is also possible - no it's enabled generally. Don't really know what is better. Probably not enabling it and a readme entry with a example portlet.xml is the best way.